src/components/ha-form/ha-form.ts

```diff
@@ -45,6 +47,11 @@ export interface HaFormSelectSchema extends HaFormBaseSchema {
   options?: string[];
 }
 
+export interface HaFormMultiSelectSchema extends HaFormBaseSchema {
+  type: "multi_select";
```

Comment:
I wonder if this should be it's own interface or that the select schema should get a `multi=true` option passed in via the description ? 

```python
vol.Schema({
  vol.Required("something", description={"multi": True}): vol.In(options_dict)
})
```

Reply:
I think that this comment is how we can get it into the backend today.